src/jquery.smooth-scroll.js

```diff
@@ -212,12 +212,9 @@
     // automatically calculate the speed of the scroll based on distance / coefficient
     if (speed === 'auto') {
 
-      // $scroller.scrollTop() is position before scroll, aniProps[scrollDir] is position after
+      // $scroller[scrollDir]() is position before scroll, aniProps[scrollDir] is position after
       // When delta is greater, speed will be greater.
-      delta = aniProps[scrollDir] - $scroller.scrollTop();
-      if(delta < 0) {
-        delta *= -1;
-      }
+      delta = Math.abs(aniProps[scrollDir] - $scroller[scrollDir]());
```

Comment:
I suppose this change is fine. In micro-benchmarks that I saw, at least a year or so ago, it was faster to do the "test and multiply by -1" thing. But since the change is in here already, I'm not going to make you revert it.